dxw/pagination
# Fix deprecation of dynamic properties for PHP8.2

PHP 8 has introduced stricter rules regarding dynamic properties in classes. 

Previously, this component used dynamic properties, which are no longer supported in PHP 8. 
As a result, sites using this dependency were encountering deprecation errors when they upgraded to PHP 8.

TODO:
- [ ] Create `CHANGELOG.md` 
- [ ] Create scripts to rule them all

## How to Test
1. Run a site that uses the dependency, such as Bikeshed or Advisories
2. Perform a search on the site
3. Confirm you are seeing the deprecation errors
4. Run `composer require dxw/pagination:dev-fix/deprecation-error-related-to-creation-of-dynamic-properties --update-with-dependencies` 
5. Refresh the page and confirm the errors are no longer present
